README.md

```diff
@@ -34,24 +34,31 @@ To use the webgazer script in the head of an HTML file add the `async` tag to en
 
 Examples of how WebGazer.js works can be found [here](https://webgazer.cs.brown.edu/#examples).
 
-### How to run the Index HTML
+### How to run the Example HTML files
 
-The Index HTML File provides the additional features of an integrated action bar that provides the functionality of a calibration system, accuracy measurements and an informative "help" module.
+Within the /www directory there are two example HTML files:
 
-This is a good example that shows how WebGazer can be used.
+  * `calibration.html`: This example includes additional user feedback, such as a 9-point calibration sequence, accuracy measurements and an informative help module.
+  * `collision.html`: This example contains a game where the user can move an orange ball with their eyes, which in turn collides with blue balls.
 
-1. Clone and build the repository using the steps listed above (including downloading NodeJS and running `npm install`)
-2. Run the index.html file as a server by running the command `browser-sync start --server --files "*"` in the WebGazer directory
+To run the example files as a server:
+
+	# Clone the repository and download NodeJS using the steps listed above
+	# Move into the www directory and download the additional dependencies
+	cd www
+	npm install
+	# Run the webpage index.html as a server
+	browser-sync start --server --files "*"
```

Comment:
Because the index html is under the www folder, this command line won't really work. Instead it should be changed to `browser-sync start --server --files --startPath "www/index.html"

Reply:
Does this not work for you? It works for me if you are in the www folder. Alex will copy across only the www folder onto the server.

Reply:
No it does not work for me if i use the original line if I am in the www folder. I have encountered this problem when I worked on this merge branch before. For some reason, browsersync cannot reference to the file in the parent directory. To solve this problem, I use the new command line when I am in the parent directory and it works

Reply:
But it should not be accessing anything in the parent directory anymore. Have you followed the new instructions?